src/vm/fcall.h

```diff
@@ -378,7 +378,7 @@ LPVOID __FCThrowArgument(LPVOID me, enum RuntimeExceptionKind reKind, LPCWSTR ar
 #ifdef __GNUC__
 #define F_CALL_CONV __attribute__((cdecl, regparm(3)))
 
-// GCC fastcall convention (simulated via stdcall) is different from MSVC fastcall convention. GCC can use up
+// GCC FCALL convention (simulated via cdecl) is different from MSVC FCALL convention. GCC can use up
```

Comment:
I'd suggest adding the regparm(3) here too, as the combination of those two is what gets the calling convention.
```suggestion
// GCC FCALL convention (simulated via cdecl, regparm(3)) is different from MSVC FCALL convention. GCC can use up
```